src/main/java/io/temporal/internal/sync/TestActivityEnvironmentInternal.java

```diff
@@ -115,6 +114,7 @@ public TestActivityEnvironmentInternal(TestEnvironmentOptions options) {
             WorkflowServiceStubsOptions.newBuilder()
                 .setChannel(channel)
                 .setMetricsScope(options.getMetricsScope())
+                .setQueryRpcTimeout(600000)
```

Comment:
what is this timeout? is it in ms? should we have a constant/take a duration here?

Reply:
Done.